### ISSUE TITLE
git failing to pull due to dubious ownership in repository

### ISSUE THREAD

**Issue by jshank** [Jafner/5etools-docker#10]
I'm unable to update the content since 1.172.3 due to git mistrusting the directory ownership.
```
SOURCE=GITHUB
5etools-docker    |  === Using GitHub mirror at https://github.com/5etools-mirror-1/5etools-mirror-1.github.io.git
5etools-docker    |  === Using existing git repository
5etools-docker    |  === Pulling from GitHub with images... (This will take a while)
5etools-docker    | fatal: detected dubious ownership in repository at '/usr/local/apache2/htdocs'
5etools-docker    | To add an exception for this directory, call:
5etools-docker    | 
5etools-docker    |     git config --global --add safe.directory /usr/local/apache2/htdocs
```

**Comment by Jafner** [comment]
I encountered this as well. I believe it started popping up recently because the previous build of this image used an earlier version of Git without the (relatively) new [CVE-2022-24765](https://github.blog/2022-04-12-git-security-vulnerability-announced/#cve-2022-24765) mitigation.  

Fix on the way.

**Comment by Jafner** [comment]
In the meantime, I was able to work around the issue with the fix in the current version ([852d01d](https://github.com/Jafner/5etools-docker/commit/852d01d98d8e80b39d82c0d746b954275a7ef772)) by deleting the repository from the mounted location and restarting the container. The fix adds the internal directory for the repo the image's global git safe directory config, but it only runs when the script is *not using an existing directory*.

**Comment by Jafner** [comment]
Closing as stale. Feel free to re-open if you can reproduce the issue with a fresh repo on the current version of the image.
